scripts/build/Jenkins/Jenkinsfile

```diff
@@ -481,12 +485,12 @@ def ExecuteControlFile(controlFile, repoName, processed_repos, o3de_project, def
         }
     }
     //queue enable all gems specified in the control file
-    if(controlFile.ENABLE_GEMS) {
+    if(controlFile.ENABLE_GEMS && o3de_project) {
         controlFile.ENABLE_GEMS.each { gemName ->
             if(o3de_project) {
-                defer_enable_gems.add([COMMAND: "${workspace}/${ENGINE_REPOSITORY_NAME}/scripts/o3de${ext} enable-gem --gem-path ${workspace}/${repoName}/${gemName} --project-name ${o3de_project} || echo>nul", ECHO: "Enabling ${workspace}/${repoName}/${gemName} for ${o3de_project}"])
+                defer_enable_gems.add([COMMAND: "${workspace}/${ENGINE_REPOSITORY_NAME}/scripts/o3de${ext} enable-gem --gem-path ${workspace}/${repoName}/${gemName} --project-name ${o3de_project}", ECHO: "Enabling ${workspace}/${repoName}/${gemName} for ${o3de_project}"])
```

Comment:
The inner `if(o3de_project)` check should no longer be needed.

Reply:
As noted in the previous comment, removing the inner check results in more runs (OpenVk and OpenXR running twice)